src/providers/ad/ad_common.c

```diff
@@ -977,17 +957,17 @@ ad_resolve_callback(void *private_data, struct fo_server *server)
     if ((sdata == NULL || sdata->gc == false) &&
         service->krb5_service->write_kdcinfo) {
         /* Write krb5 info files */
-        safe_address = sss_escape_ip_address(tmp_ctx,
-                                            srvaddr->family,
-                                            address);
-        if (safe_address == NULL) {
+        safe_addr_list[0] = sss_escape_ip_address(tmp_ctx,
+                                                  srvaddr->family,
```

Comment:
NIT.tab space.

Reply:
I don't understand, I don't see any tab there. And my .vimrc is configured to show tabs with `set list listchars=tab:»` and to for SSSD I expand tabs to spaces with `set expandtabs`

Reply:
I do not see tabs either. @amitkumar50, is there a chance you interpreted the change in whitespace characters indicated by github with a slightly darker color as tab?.